docs/src/main/paradox/server.md

```diff
@@ -141,6 +141,10 @@ That service can then be handled by an Akka HTTP server via the generated `Greet
 which is a @scala[partial ]function from `HttpRequest` to @scala[`Future[HttpResponse]`]@java[`CompletionStage<HttpResponse>`].
 @scala[The partial function should be made total before giving it to Akka HTTP by for example providing 404 as as default response].
 
+The server will run one instance of the implementation and that is then shared between requests,
+this mean that it must be thread safe. In the sample above there is no mutable state, for more about safely implementing
+servers with state see @link-ref-how-do-you-do-it?[stateful]
```

Comment:
heh I guess `[stateful](#Stateful-services)`? Not entirely sure.

Reply:
Yes, that did the trick. Thanks!